### PR TITLE
Motion detection implementation

### PR DESCRIPTION
**Pull request by nik0**
Execute /system/sdcard/scripts/detectionOn.sh script when something have been detected
Execute /system/sdcard/scripts/detectionOff.sf script when nothing more to detect

### CODE REVIEW

File: v4l2rtspserver-tools/setconf.cpp [hunk 1]
```diff
@@ -3,64 +3,149 @@
...
+    fprintf(stderr, "\t\t'4 for Blue\n");
+    fprintf(stderr, "\t\t'5' for Cyan\n");
+    fprintf(stderr, "\t\t'6' for Yellow\n");
+    fprintf(stderr, "\t\t'7' for Purple\n");
+
+    fprintf(stderr, "\t'x' OSD position Y pos is set to VALUE\n");
+    fprintf(stderr, "\t'p' OSD space between char is set to VALUE (can be negative)\n");
+    fprintf(stderr, "\t'w' fixed text width (0 variable, 1 fixed)\n");
+
+    fprintf(stderr, "\t'm' motion sensibility (0 to 4) -1 to deactivate motion\n");
```
Comment (jmtatsch) [inline v4l2rtspserver-tools/setconf.cpp]: Love your contribution. However, can the parameter be called sensitivity please?
The basic distinction is that sensitivity is a fairly universal concept for “readily reacts to someone or something” while sensibility basically means “how practical as a remedy or strategy [a person or plan] is.”

Comment (nik0) [reply v4l2rtspserver-tools/setconf.cpp]: Good catch ! I didn't realize that ...
Corrected !

**Comment by EliasKotlyar** [comment]
Hello @nik0 ,

Many thanks for your efforts. I had no time before, but now i have merged it. It looks fine from the code-side.

Greatings
Elias

**Comment by nik0** [comment]
No problem...
Do you plan to compile and put the binaries to “Hack” repo ?

**Pull request merged**